packages/dev/core/src/Materials/Textures/videoTexture.ts

```diff
@@ -192,7 +192,10 @@ export class VideoTexture extends Texture {
         this._currentSrc = src;
         this.name = name || this._getName(src);
         this.video = this._getVideo(src);
-        this._externalTexture = this._engine?.createExternalTexture(this.video) ?? null;
+        this._externalTexture = (this._engine?.createExternalTexture && this._engine.createExternalTexture(this.video)) ?? null;

```

Comment:
`this._externalTexture === null` is valid, it's what you will get in WebGL. Only WebGPU uses external textures and will return a non null value.